internal/metrics/metricsManager.go

```diff
@@ -377,8 +389,10 @@ func (m *MetricsManager) ReportHostInstallationMetrics(log logrus.FieldLogger, c
 			diskType = boot.DriveType
 		}
 		switch currentStage {
-		case models.HostStageDone, models.HostStageFailed:
-			m.handleHostInstallationComplete(log, clusterVersion, clusterID, emailDomain, roleStr, hwVendor, hwProduct, diskType, installationStageStr, h)
+		case models.HostStageDone:
+		case models.HostStageFailed:
```

Comment:
You changed the logic here, the old implementation will enter the case if one of the conditions is true, the new implementation have an empty case for `HostStageDone` (won't do anything in this case).

This is what you meant?
https://play.golang.org/p/eMPuDQ2GFqb

Reply:
Nope. Will Fix